src/commands/plugins.rs

```diff
@@ -320,12 +320,13 @@ impl Upgrade {
     }
 }
 
-/// List available or installed plugins.
+/// List installed plugins.
```

Comment:
It looks like the following line needs to be updated as well: https://github.com/fermyon/spin/blob/c3c1de2fd0fcd33aab6d345e4273d2c526c77222/src/commands/plugins.rs#L29